experimental/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { TypedEventEmitter } from "@fluidframework/common-utils";
+import { Container } from "@fluidframework/container-loader";
+import { IFluidLoadable } from "@fluidframework/core-interfaces";
+import { IEvent } from "@fluidframework/common-definitions";
+import { LoadableObjectClass } from "./types";
+import { RootDataObject } from "./rootDataObject";
+
+interface IFluidContainerEvents extends IEvent {
+    (event: "connected", listener: (clientId: string) => void): void;
+    (event: "dispose" | "disconnected", listener: () => void): void;
+}
+
+interface IFluidContainer {
+    close: Container["close"];
+    closed: Container["closed"];
+    connected: Container["connected"];
+    initialObjects: RootDataObject["initialObjects"];
+}
+
+export class FluidContainer extends TypedEventEmitter<IFluidContainerEvents> implements IFluidContainer {
+    public constructor(
+        private readonly container: Container,
+        private readonly rootDataObject: RootDataObject,
+    ) {
+            super();
+            container.on("connected", (id: string) =>  this.emit("connected", id));
+            container.on("dispose", () =>  this.emit("dispose"));
+            container.on("disconnected", () =>  this.emit("disconnected"));
+    }
+
+    public static async load(container: Container): Promise<FluidContainer> {
+        const rootDataObject = (await container.request({ url: "/" })).value;
+        return new FluidContainer(container, rootDataObject);
+    }
```

Comment:
This method bakes a lot of assumptions in -- that the container has a default request handler for "/", that the default request handler will return a `RootDataObject`, that the request won't error for other reasons,

Since these are assumptions that `FluidContainer` can't necessarily make safely, we should not include `load()` as a member.  If you still want a convenience function I'd recommend a standalone helper function (and also be sure to validate the request response), probably living next to the `DOProviderContainerRuntimeFactory` which is the source of these assumptions (and if the assumptions ever change, both need to be updated in sync).